jobs/base_model_validator_test.py

```diff
@@ -169,3 +173,51 @@ def test_validate_model_id(self):
                 beam_testing_util.equal_to([
                     errors.ModelInvalidIdError(invalid_id_model)
                 ]))
+
+
+class ValidatePostCommitIsPrivateTests(BaseModelValidatorTests):
+    def test_validate_post_commit_is_private_when_status_is_public(self):
+        with pipeline.TestPipeline(runner=direct_runner.DirectRunner()) as p:
+            invalid_commit_status = MockCommitLogEntryModel(
```

Comment:
Please use the real model and delete the mock.

Reply:
Done